<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welch</class>
 <widget class="QMainWindow" name="welch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>senales cerebrales</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="campo_grafico" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>421</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_cargar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>cargar señal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tiempoinicial">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tiempofinal">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo inicial</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo final</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="canales">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Canales</string>
    </property>
   </widget>
   <widget class="QLabel" name="numero_canales">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>240</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>240</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Número de canales:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_atras">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_adelante">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_disminuir">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Disminuir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_w">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Aumentar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="welch_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No se sobreponen</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Se sobreponen al 50%</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Se sobreponen el doble</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
